packages/package-tests/typescript/tsconfig.json

```diff
@@ -3,6 +3,7 @@
     "baseUrl": "./",
     "sourceRoot": "./",
     "compilerOptions": {
+        "skipLibCheck": true,
```

Comment:
@darrellwarde The errors that caused the typescript package-test to fail all occurred in the `node_modules` folder. Using `skipLibCheck` ([docs](https://www.typescriptlang.org/tsconfig#skipLibCheck)) solves the issue. Is there anything against this approach you can think of?